subjects/java/projects/sharpen-it/README.md

```diff
@@ -0,0 +1,61 @@
+## Sharpen-It
+
+### Objectives
+
+In this project, your task is to finalize the provided environment for your project, applying coding best practices to enhance clarity and cleanliness.
+
+### Instructions
+
+You have to download this [Environment](./env.zip), then you will find two folders, one for Jenkins and another for SonarQube. Your goal is to finish the given infrastructure and make the environment functional and compliant with the below requirements.
+
+#### Jenkins
+
+- Build and run the Jenkins Docker image.
+
+- Install missing plugins.
+
+- Set up credentials to access your repository.
+
+- Fix the provided pipeline to build and test the back-end and front-end
+
+- Complete and rectify the configuration for deployment.
+
+- Finalize the configuration for the rollback strategy.
+
+- Configure your repository to trigger the CI/CD pipeline for every approved pull request (PR).
+
+- Configure Jenkins to automatically merge a new PR if the build is successful during deployment.
+
+> 💡 Feel free to use GitHub or GitLab for environment configuration.
+
+#### SonarQube
+
+- Run the provided Docker Compose for SonarQube.
+
+- Complete the configuration to integrate SonarQube with your GitHub repository.
```

Comment:
Linked tp the previous comment, at line 29, if they're free to use GitHub or GitLab, shouldn't this be more "open"? Or should we just stick with GitHub, for example, to keep things simple?